README.md

```diff
@@ -1,70 +1,84 @@
     │    |
-    │    ├── ARCH_X        <- Directory for architecture X
+    │    ├── ARCH_X        <- Directory for architecture X.
     |        ├── ARCH_X_001.h5
     |        ├── ARCH_X_001.json
     |        │
-    |        └── ARCH_X_002  
-    │  
+    |        └── ARCH_X_002
+    │
     ├── src                 <- Source code to use in this project.
     |    |
-    │    ├── __init__.py    <- Makes src a Python module
+    │    ├── __init__.py    <- Makes src a Python module.
     |    |
-    │    ├── utils.py       <- General utility functions   
+    │    ├── utils.py       <- General utility functions.
+    |    |
+    │    ├── main.py        <- Main code for running an API/application.
+    │    │
+    │    ├── app            <- Code used in deployment.
+    │    │
+    │    ├── preparation    <- Data preparation. Retrieves data and puts in data/raw/, transforms to data/processed/ using src/processing/.
     │    │
-    │    ├── preparation    <- Data ingestion and retrieval. Retrieves data and puts in data/raw
+    │    ├── processing     <- Data transformation and pre-processing.
     │    │
-    │    ├── processing     <- Data transformation and pre-processing. Processes data from data/raw -> data/processed
+    │    ├── training       <- Code for training and evaluating models.
     │    │
-    │    └── modelling      <- Model classes, training and evaluation scripts
+    │    └── modelling      <- Model classes.
     │
-    ├── test                <- Code used for testing src/
+    ├── test                <- Code used for testing src/.
     │
-    │    
-    ├── requirements.txt    <- The requirements file for reproducing the analysis environment
+    ├── requirements.txt    <- The requirements file for reproducing the environment.
     │
-    ├── setup.py            <- Makes project pip installable (pip install -e .) so src can be imported
+    ├── setup.py            <- Makes project pip installable so src can be imported.
 
 ## How to work - phases
 
-#### POC phase
+### POC phase
 
 In this phase, you may use some kaggle dataset or a small collected dataset. Focus should be on displaying potential and not building a whole sophisticated system.
 
-Work in notebooks and focus on training a model on your initial dataset and see if there's any potential.
+### Post-POC phase
 
-#### Post-POC phase
+Now it's time to explore the real data. Start by working in notebooks/eda to really understand the data you're working with.
 
-Now it's time to explore the real data. Start by working in notebooks/eda to really understand the data you're working with. Expand into all the other unused directories as the code you write in the notebooks becomes robust.
+When the code you write in the notebooks becomes robust, turn it into functions in src/. If the notebook is still useful for visualization or demonstration purposes, import these functions from src/, otherwise remove the notebook.
```

Comment:
Vad tycker vi om att säga att som standard ta bort notebooken? Jag gillar personligen att ha kvar, kan vara användbart att ha som sandbox, eller ifall att man vill gå tillbaka till något gammalt

Reply:
Anledningen till den ändringen är att jag inte vill ha samma kod på två ställen, som det står: om notebooken fortfarande är användbar, importera funktionerna istället, och om man vill göra nya experiment får man kopiera in koden för funktionen på nytt, ok?

Reply:
Låter bra!